### DISCUSSION TITLE
How can I get --hot to pick up on changes in resources processed by a custom loader?

### DISCUSSION THREAD

**Discussion by borkdude** [oven-sh/bun#5374] [Q&A] [answered]
This is an example project:

https://github.com/borkdude/squint-bun-cloudflare

When I run `bun --hot src/index.js` and make changes to `index.js`, changes are picked up beautifully.
However, `index.js` loads `"./index.cljs"` and process the `.cljs` file using a custom loader (see `squintPlugin.js`). When I make changes to `index.cljs` I'd like `--hot` to pick up changes from that file too. Is this possible?

**Comment by sirenkovladd** [comment, marked as answer]
I think it is duplicate of https://github.com/oven-sh/bun/issues/4689

**Reply by borkdude** [reply]
Indeed, thank you. I'll start watching that issue.
